### DISCUSSION TITLE
Şehir Sayısı

### DISCUSSION THREAD

**Discussion by HabilCcabbar** [sonsuzus/sonsuzus#341] [Soru Cevap]
James bir tatile çıkmaya karar verir. Türkiye kültürü ile ilgilendiğinden Türkiye'ye gitmeye karar verir. Gidebileceği şehirleri bir listeye yazar. Listedeki şehir sayısına **n** diyelim. Sonra karısına listeyi gösterir ve karısına da aynı şeyi yapmasını söyler. Karısının herhangi bir miktarda ve herhangi bir şehri yazabileceğini söyler. Karısı ise bir gün müddet ister. 

Bundan sonra James oğlunu arar ve her şeyi ona anlatır. Ardından ona bir görev verir. Listedeki şehirleri kaç farklı bir şekilde sıralayabilir? Şehirler **1'den n'ye** olarak sıralanmıştır ve en Az bir şehir seçilmelidir.

Şimdi de aynı soruyu size soruyor James, Listedeki şehirleri kaç farklı bir şekilde sıralayabilirsiniz?

_GİRİŞ:_

İlk giriş satırı **t** sayısında test senaryosu içermektedir.
**t** satırları, **n** sayısını içerir.

_ÇIKIŞ:_

Çıkış **t** satırları içerir. her satır o test senaryosunun cevabını içermelidir. Cevap çok büyük olabileceğinden dolayı, **10^9+7** modeline uygun olmalıdır.

_KISITLAMALAR:_

**1<=t<=100000**
**1<=n<=10^12**

_Örnek Giriş:_                                            
2                                                                
2                                                                
1

_Örnek Çıkış:_
3
1

Orijinal Soru: https://www.hackerearth.com/practice/math/number-theory/basic-number-theory-1/practice-problems/algorithm/selection-of-cities-2/

**Comment by nilbuyukkinaci** [comment]
```c
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int t, n[100005];

    cin >> t;

    for (int i = 0; i < t; i++)
        cin >> n[i];

    for (int i = 0; i < t; i++)
        cout << pow(2, n[i]) - 1 << endl;
}
```
